src/pydata_sphinx_theme/assets/styles/extensions/_togglebutton.scss

```diff
@@ -3,10 +3,69 @@
  */
 
 .bd-content {
+  @mixin chevron-down {
+    .toggle-chevron-right {
+      transform: rotate(90deg); // point chevron down
+      transition: none; // match non-animated behavior of other chevrons on site
+    }
+  }
+
+  @mixin chevron-up {
+    .toggle-chevron-right {
+      transform: rotate(-90deg); // point chevron down
+      transition: none; // match non-animated behavior of other chevrons on site
+    }
+  }
```

Comment:
both of these mixins have the same comment `point chevron down` but they're rotations in opposite directions. Seems like one of the comments must be wrong